htdocs/gallery.php

```diff
@@ -23,8 +23,14 @@
     $attributes[] = $attributes_map[$gallery_title]['attribute'];
     $attributes[] = $attributes_map[$gallery_sortby]['attribute'];
 
+    # Search for users in group
+    if (isset($_GET['dn'])) {
+        $gallery_user_filter = substr($ldap_user_filter, 0, -1).'(memberOf='.$_GET['dn'].'))';
```

Comment:
What if we defined gallery_user_filter in configuration? Here you are wiping this configuration value.